### PR TITLE
update default value of agent-image-name org to open-cluster-management

### PR DESCRIPTION
**Pull request by TheRealHaoLiu**
Signed-off-by: Hao Liu <haoli@redhat.com>

### CODE REVIEW

File: cmd/manager/main.go [hunk 1]
```diff
@@ -57,7 +57,7 @@ func main() {
 	var addonAgentImageName string
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":38080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":38081", "The address the probe endpoint binds to.")
-	flag.StringVar(&addonAgentImageName, "agent-image-name", "yue9944882/managed-serviceaccount-addon-agent:v0.0.0",
+	flag.StringVar(&addonAgentImageName, "agent-image-name", "open-cluster-maangement/managed-serviceaccount-addon-agent:v0.0.0",
```
Comment (qiujian16) [inline cmd/manager/main.go:60]: I think it should be quay.io/open-cluster-management?

Comment (yue9944882) [reply cmd/manager/main.go:60]: https://github.com/open-cluster-management-io/managed-serviceaccount/blob/741d9c369e6822c98591c91b9fe86e571d44a5e5/charts/managed-serviceaccount/templates/manager-deployment.yaml#L24

the default value doesnt take effect in helm installation tho

Comment (TheRealHaoLiu) [reply cmd/manager/main.go]: oops... fix the typo

File: cmd/manager/main.go [hunk 2]
```diff
@@ -57,7 +57,7 @@ func main() {
 	var addonAgentImageName string
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":38080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":38081", "The address the probe endpoint binds to.")
-	flag.StringVar(&addonAgentImageName, "agent-image-name", "yue9944882/managed-serviceaccount-addon-agent:v0.0.0",
+	flag.StringVar(&addonAgentImageName, "agent-image-name", "open-cluster-management/managed-serviceaccount-addon-agent:v0.0.0",
```
Comment (yue9944882) [inline cmd/manager/main.go:60]: how about `quay.io/open-cluster-management/managed-serviceaccount:latest`?

**Pull request merged**